src/components/testCaseList/TestCase.tsx

```diff
@@ -127,12 +135,16 @@ const TestCase = ({
           />
         </StyledCell>
 
-        {status === "NA" && <td>{status}</td>}
-        {status !== "NA" && (
-          <td>
-            <Chip label={status} color={statusColor} />
-          </td>
-        )}
+        {/*{status === "NA" && (*/}
+        {/*  <td>*/}
+        {/*    <Chip label={status} color="primary" />*/}
+        {/*  </td>*/}
+        {/*)}*/}
+        {/*{status !== "NA" && (*/}
+        <td>
+          <Chip label={status} color={statusColor} />
+        </td>
+        {/*)}*/}
```

Comment:
I guess you forgot to remove the old code

Reply:
will remove in subsequent PR